Jint/Native/Date/DateConstructor.cs

```diff
@@ -87,9 +87,26 @@ protected override void Initialize()
             SetProperties(properties);
         }
 
+        /// <summary>
+        /// https://tc39.es/ecma262/#sec-date.parse
+        /// </summary>
         private JsValue Parse(JsValue thisObj, JsValue[] arguments)
         {
             var date = TypeConverter.ToString(arguments.At(0));
+            var negative = date.StartsWith("-");
```

Comment:
wtf

Reply:
Trial error horror I guess, I will be having only things related to test in this PR when merged, will exclude Jint changes.